client_databases.go

```diff
@@ -62,6 +62,13 @@ type CreateDatabaseOptions struct {
 	Options CreateDatabaseDefaultOptions `json:"options,omitempty"`
 }
 
+type DatabaseReplicationVersion string
```

Comment:
```suggestion
// DatabaseReplicationVersion defines replication protocol version to use for this database
// Available since ArangoDB version 3.11
// Note: this feature is still considered experimental and should not be used in production
type DatabaseReplicationVersion string
```